src/Forms/EffectKaraoke.cs

```diff
@@ -290,8 +289,8 @@ private void MakeAnimation()
             if (numericUpDownDelay.Value > 0)
             {
                 startMilliseconds = index * stepsLength;
-                startMilliseconds += _paragraph.StartTime.TotalMilliseconds;
-                endMilliseconds = _paragraph.EndTime.TotalMilliseconds;
+                startMilliseconds += totalMilliseconds;
+                endMilliseconds = totalMilliseconds;

```

Comment:
`totalMilliseconds = _paragraph.StartTime.TotalMilliseconds`
and
 `endMilliseconds = totalMilliseconds`
==>
 `endMilliseconds = _paragraph.StartTime.TotalMilliseconds`

That would not be the same as the original
 `endMilliseconds = _paragraph.EndTime.TotalMilliseconds`